### PR TITLE
feat: Added english docs to the repo

### PR DESCRIPTION
**Pull request by ArslanYM**

### CODE REVIEW

**Comment by ArslanYM** [comment]
Closes #2465.

**Review by chanlee — APPROVED** [review]

File: README-en.md [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+                id: String // Scene ID. Unique.
+            }
+        ]
+    },
+});
+```
+
+### Sprite, Picture, Sound schema
+
+### 스프라이트
```
Comment (chanlee) [inline README-en.md:345]: Sprites

File: README-en.md [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+                }],
+                sounds: [{ // list of sounds
+                    name: String, // name
+                    fileurl: String, // Sound file URL
+                    duration: Number // play time. (in seconds)
+                }]
+            })
+```
+
+#### 모양
```
Comment (chanlee) [inline README-en.md:365]: Pictures

File: README-en.md [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+                    name: String, // shape name
+                    fileurl: String, // shape image
+                    dimension: { // shape size
+                        width: Number,
+                        height: Number
+                    }
+	})
+```
+
+#### 소리
```
Comment (chanlee) [inline README-en.md:377]: Sounds

**Comment by ArslanYM** [comment]
@chanlee I have made the requested changes to the file. Please review.

**Comment by ArslanYM** [comment]
Are there any other changes required? @chanlee

**Comment by Tnks2U** [comment]
@ArslanYM 
The content doesn't seem to be a problem.
By the way, is gitpod.yml necessary?

**Comment by Tnks2U** [comment]
@ArslanYM 
gitpod.yml seems unnecessary, so I'd like to exclude that change.
If you give me a confirmation comment, I will merge immediately.

**Comment by ArslanYM** [comment]
@Tnks2U No gitpod.yml is not necessary. You can proceed with merging. Thanks

**Review by Tnks2U — APPROVED** [review]

**Comment by Tnks2U** [comment]
@ArslanYM 
Your PR has been merged. Thank you for your concern and effort!

**Pull request merged**